src/app/storify/storify-explore/storify-explore.component.scss

```diff
@@ -0,0 +1,6 @@
+.button-container {
```

Comment:
on smaller screens, the button gets in the way of the cards... I'd place it in the top-right corner instead of bottom-right... and I'd make the margins absolute, looks kinda weird if the button is further away on bigger screens than small ones